google_project/main.tf

```diff
@@ -29,4 +29,34 @@ resource "google_project_service" "project" {
   project            = local.project_id
   service            = each.key
   disable_on_destroy = false
-}
\ No newline at end of file
+}
+
+resource "google_project_iam_audit_config" "data_access_high" {
+  count = var.risk_level == "high" ? 1 : 0
+
+  project = local.project_id
+  service = "allServices"
+  audit_log_config {
+    log_type = "ADMIN_READ"
+  }
+  audit_log_config {
+    log_type = "DATA_READ"
+  }
+  audit_log_config {
+    log_type = "DATA_WRITE"
+  }
+}
```

Comment:
This resource is already available on main, we just need to update the branch to include this.